sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/implementation/SnapshotImpl.java

```diff
@@ -321,4 +384,56 @@ private String constructStorageAccountId(String vhdUrl) {
                     new IllegalArgumentException(String.format("%s is not valid URI of a blob to import.", vhdUrl)));
         }
     }
+
+    private Mono<Snapshot> waitCopyStartToFinish() {
+        final long startTime = System.currentTimeMillis();
```

Comment:
usually we should use monotonic time instead of wall time.

Reply:
changed